Library/MeterImage.cpp

```diff
@@ -55,14 +54,28 @@ void MeterImage::Initialize()
 ** Loads the image from disk
 **
 */
-void MeterImage::LoadImage(const std::wstring& imageName, bool bLoadAlways)
+void MeterImage::LoadImageFromFile(const std::wstring& imageName, bool bLoadAlways)
 {
-	m_Image.LoadImage(imageName);
+	m_Image.LoadImageFromFile(imageName);
+	CalcImageDimensions();
+}
 
+void MeterImage::LoadImageFromPluginMeasure(MeasurePlugin *mPlugin)
+{
+	m_Image.LoadImageFromPluginMeasure(mPlugin);

```

Comment:
Actually i didn't want to add this function to `GeneralImage` class, because we will only use it in `MeterImage`. But some members (like `m_Bitmap`) were set to private, and i didn't know how to access them.